rbac/management/role/definer.py

```diff
@@ -116,19 +120,19 @@ def _make_role(data, dual_write_handler):
     return role
 
 
-def _update_or_create_roles(roles, dual_write_handler):
+def _update_or_create_roles(roles, dual_write_handler, force_create_relationships=False):
     """Update or create roles from list."""
     current_role_ids = set()
     for role_json in roles:
         try:
-            role = _make_role(role_json, dual_write_handler)
+            role = _make_role(role_json, dual_write_handler, force_create_relationships)
```

Comment:
```
if force_create_relationships
    dual_write_handler = new()
    dual_write_handler.replicated_new_role(role)
else
    role = _make_role(role_json, dual_write_handler)
end
```

Reply:
Okay the problem I see with this is that we don't actually have the role here to replicate only the json representation. We would still need to get the role like we do in `_make_role`

Reply:
I see! so I am fine with as it is now, thanks!